<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>599</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>599</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolTip">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
	color: rgb(0, 255, 255);
	background-color:  rgb(0, 135, 255);
	border: 2px solid rgb(0, 204, 100);
	border-radius: 5px;
}
QPushButton:pressed { background-color:rgb(35, 200, 100) }

#centralwidget
{
	background:url(&quot;C:/Users/KB/Desktop/sss/python_work/pythonProject/배경11.png&quot;)
	
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>11</number>
    </property>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>21</number>
      </property>
      <property name="topMargin">
       <number>25</number>
      </property>
      <property name="rightMargin">
       <number>53</number>
      </property>
      <property name="bottomMargin">
       <number>60</number>
      </property>
      <item row="0" column="0">
       <widget class="QTextBrowser" name="list">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="openLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>66</number>
        </property>
        <property name="rightMargin">
         <number>41</number>
        </property>
        <property name="bottomMargin">
         <number>60</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>45</width>
            <height>37</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>고객 정보 관리</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="minimumSize">
           <size>
            <width>45</width>
            <height>37</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>고객 이메일 리스트</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="minimumSize">
           <size>
            <width>45</width>
            <height>37</height>
           </size>
          </property>
          <property name="text">
           <string>상품 검색</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_4">
          <property name="minimumSize">
           <size>
            <width>45</width>
            <height>37</height>
           </size>
          </property>
          <property name="text">
           <string>이메일 보내기</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_5">
          <property name="minimumSize">
           <size>
            <width>45</width>
            <height>37</height>
           </size>
          </property>
          <property name="text">
           <string>엑셀</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_6">
          <property name="minimumSize">
           <size>
            <width>45</width>
            <height>37</height>
           </size>
          </property>
          <property name="text">
           <string>어플 사용 설명서</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
